.gitignore

```diff
@@ -0,0 +1,9 @@
+*.egg-info
+.coverage
+.ipynb_checkpoints/
+.mypy_cache/
+.pytest_cache/
+.vscode/
+build/
+xroms/_version.py
```

Comment:
This is the only mandatory one b/c we don't want to commit the version file. `setuptools_scm` will handle that for us.